app/controllers/user/birth_records_controller.rb

```diff
@@ -1,24 +1,35 @@
 # frozen_string_literal: true
 
 class User::BirthRecordsController < User::BaseController
+  skip_after_action :verify_authorized, only: %i[query remove]
+  # Remove sometimes has no birth_record to authorize, and should silently fail
+  after_action :verify_policy_scoped, only: :query
+
   # GET
   def index
-    @birth_records = current_user.birth_records
+    @birth_records = policy_scope(current_user.birth_records)
   end
 
   # GET
   def show
     @birth_record = current_user.birth_records.find_by!(params.permit(:id))
+    authorize @birth_record
     @shares = @birth_record.shares.where(user: current_user)
   end
 
+  # GET
+  def find
```

Comment:
Unclear about why this method was deleted in ee94dfdb39bb7286f61c80376b004f2e456318ab

Reply:
.... bad git merging. I'll delete again